x/stakeibc/keeper/icacallbacks.go

```diff
@@ -46,7 +48,9 @@ func (c ICACallbacks) AddICACallback(id string, fn interface{}) icacallbackstype
 }
 
 func (c ICACallbacks) RegisterICACallbacks() icacallbackstypes.ICACallbackHandler {
-	a := c.AddICACallback("delegate", ICACallback(DelegateCallback))
+	a := c.
+		AddICACallback("delegate", ICACallback(DelegateCallback)).
```

Comment:
should we store the callback key's as constants at the top of this file so we can access them when we call SubmitTx?

Reply:
done